packages/rich-text/src/component/index.js

```diff
@@ -325,8 +325,32 @@ function RichText(
 			multilineTag,
 			preserveWhiteSpace,
 		} );
-		event.clipboardData.setData( 'text/plain', plainText );
-		event.clipboardData.setData( 'text/html', html );
+
+		const isLi = /<li>.*<\/li>/gi;
+		const foundLi = html.match( isLi );
```

Comment:
it is better to use a boolean value here with proper semantic:
`const hasLi = html.match( isLi )?.length > 0;`

Reply:
I changed my code with your comment , can you give me please a link for documentation of use this question mark after ( isLi )

Reply:
It is called optional chaining and it is a native js feature https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining